lib/widget/input/form_textfield.dart

```diff
@@ -38,6 +40,14 @@ class TextInput extends BaseTextInput {
     return setters;
   }
 
+  @override
+  Map<String, Function> getters() {
+    return {
+      'maskedValue': () => _controller.inputFieldAction?.getMaskedValue(),
+      'unmaskedValue': () => _controller.inputFieldAction?.getUnmaskedValue(),
```

Comment:
Here, you want both masked and unmasked values right?

Maybe I can remove the `maskedValue` getter property (because the value getter property will return what we entered in the textfield, with masked values). Still, you want the `unmaskedValue` getter property right?